apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -156,7 +200,11 @@ const DebugConsole = connect(
       <div
         id="debug-console"
         className={classes}
-        style={this.props.style}
+        style={{
+          display: 'flex',
+          flexDirection: 'column',
+          ...this.props.style
```

Comment:
It looks like the main problem is usage in conjunction with min-height, so I think we are safe, but I will double check.